src/apps/pulse/components/Buy/Buy.tsx

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable react/no-unused-prop-types */
```

Comment:
Pls remove these eslint ignore and remove any unused vars or property for code consistency